Makefile

```diff
@@ -0,0 +1,11 @@
+install-dev-tools:
+	composer update --working-dir=dev-tools
+
+run-php-cs-fixer:
+	dev-tools/vendor/bin/php-cs-fixer fix
+
+run-phpstan:
+	dev-tools/vendor/bin/phpstan/phpstan/phpstan analyse
+
+run-phpunit:
+	dev-tools/vendor/bin/phpunit
```

Comment:
may be allow parameters to be added at the end of command line

Reply:
Done using optional `$(ARGS)` parameter, see https://github.com/smalot/pdfparser/pull/383/commits/a65d972f0a30ed7fe5e95fdf7e307893099c3fef

If you have a better idea how to do that, please tell me. I am not a Makefile geek.